parlai/scripts/interactive.py

```diff
@@ -42,6 +42,18 @@ def setup_args(parser=None):
         default='label_candidates,text_candidates',
         help='Do not display these fields',
     )
+    parser.add_argument(
+        '--interactive-task',
+        type='bool',
+        default=True,
+        help='Create interactive version of task',
+    )
+    parser.add_argument(
+        '-t', '--task',
+        type=str,
+        default='interactive',
```

Comment:
sneaky

Reply:
it actually MIGHT be better to have this call `parser.set_default(task='interactive')` instead